.gitignore

```diff
@@ -0,0 +1,3 @@
+.idea/*
+*.iml
+target/*
```

Comment:
missing empty line in the end of the file

Reply:
done